eval_all.sh

```diff
@@ -97,22 +97,22 @@ for model in $BRMODELS; do
         OUTDIR="${OUT}/${fn}/"
         mkdir -p $OUTDIR
 
-
-
         CMD="\
             ./build/predictor $TRACE \
                 --model ${model} \
                 ${FLAGS} \
                 -w ${N_WARM} -n ${N_SIM} \
                 --output "${OUTDIR}/${model}-${POSTFIX}" \
-                > $OUTDIR/${model}-${POSTFIX}.txt 2>&1" &
+                > $OUTDIR/${model}-${POSTFIX}.txt 2>&1 &"
```

Comment:
Actually, I think we need to remove the `&`. Otherwise all processes get started in the background without waiting to complete. This would make `parallel` useless

Reply:
removed